ruby/ql/test/library-tests/frameworks/json/JsonDataFlow.ql

```diff
@@ -0,0 +1,12 @@
+/**
+ * @kind path-problem
+ */
+
+import codeql.ruby.AST
+import TestUtilities.InlineFlowTest
+import codeql.ruby.Frameworks
+import PathGraph
+
+from DataFlow::PathNode source, DataFlow::PathNode sink, DefaultValueFlowConf conf
```

Comment:
```suggestion
from DataFlow::PathNode source, DataFlow::PathNode sink, DefaultTaintFlowConf conf
```

Your inline tests are testing for taint-flow, but in this test you are outputting all the data-flows.  
Is that intentional? 
You could add a negative inline expectation about data-flow?